docs/tutorials/performance.md

```diff
@@ -65,6 +65,23 @@ The following storage drivers are listed from fastest to slowest:
 2. fuse-overlayfs
 3. vfs
 
+There is one notable exception to this speed ranking.
+Creating a container takes significantly longer with _native overlayfs_ than _fuse-overlayfs_
+when these conditions are all met:
+
+* rootless Podman is used
+* a modified UID/GID mapping is used
+* _native overlayfs_ is used
+* no container has yet been created with the specified container image and UID/GID mapping
+
+Runtime speed is not affected. Only _podman create_ and the startup phase of _podman run_ are affected.
```

Comment:
I think this should this also mention `podman build` instead of only `podman create` and `podman run` since it creates build containers in a similar way.

Reply:
@giuseppe @nalind WDYT ?